src/Components/Blazor/Blazor/test/Hosting/WebAssemblyHostBuilderTest.cs

```diff
@@ -3,160 +3,100 @@
 
 using System;
 using System.Collections.Generic;
+using System.Net.Http;
 using System.Text;
+using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.Routing;
+using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
 using Microsoft.JSInterop;
 using Xunit;
 
-namespace Microsoft.AspNetCore.Blazor.Hosting.Test
+namespace Microsoft.AspNetCore.Blazor.Hosting
 {
     public class WebAssemblyHostBuilderTest
     {
         [Fact]
-        public void HostBuilder_CanCallBuild_BuildsServices()
+        public void Build_AllowsConfiguringConfiguration()
         {
             // Arrange
             var builder = new WebAssemblyHostBuilder();
 
-            // Act
-            var host = builder.Build();
-
-            // Assert
-            Assert.NotNull(host.Services.GetService(typeof(IWebAssemblyHost)));
-        }
-
-        [Fact]
-        public void HostBuilder_CanConfigureAdditionalServices()
-        {
-            // Arrange
-            var builder = new WebAssemblyHostBuilder();
-            builder.ConfigureServices((c, s) => s.AddSingleton<string>("foo"));
-            builder.ConfigureServices((c, s) => s.AddSingleton<StringBuilder>(new StringBuilder("bar")));
+            builder.Configuration.AddInMemoryCollection(new[]
+            {
+                new KeyValuePair<string, string>("key", "value"),
+            });
 
             // Act
             var host = builder.Build();
 
             // Assert
-            Assert.Equal("foo", host.Services.GetService(typeof(string)));
-            Assert.Equal("bar", host.Services.GetService(typeof(StringBuilder)).ToString());
+            Assert.Equal("value", host.Configuration["key"]);
         }
 
-        [Fact]
-        public void HostBuilder_UseBlazorStartup_CanConfigureAdditionalServices()
+        [Fact] 
```

Comment:
Trailing whitespace!

Reply:
Feel free to implement a bot that runs `dotnet format` on the codez.